modules/external-api/src/main/java/org/opencastproject/external/endpoint/CaptureAgentsEndpoint.java

```diff
@@ -55,7 +55,7 @@
 import javax.ws.rs.core.Response;
 
 @Path("/")
-@Produces({ ApiMediaType.JSON, ApiMediaType.VERSION_1_1_0, ApiMediaType.VERSION_1_2_0, ApiMediaType.VERSION_1_3_0, ApiMediaType.VERSION_1_4_0 })
+@Produces({ ApiMediaType.JSON, ApiMediaType.VERSION_1_1_0, ApiMediaType.VERSION_1_2_0, ApiMediaType.VERSION_1_3_0, ApiMediaType.VERSION_1_4_0, ApiMediaType.VERSION_1_5_0 })
```

Comment:
Opencast doesn't have to be any enforced line length limit, but maybe it's time to break this line? This also applies to other places in this PR.

Reply:
I'll break the lines containing the API versions. The rest I'll leave as is for now, otherwise I'll still be here tomorrow. ;)

Reply:
Looks good now. Apparently I can't "resolve" these PR comments. Feel free to resolve this one yourself.